src/index.ts

```diff
@@ -357,6 +358,39 @@ async function main() {
           .finally(() => providers.destroy())
       },
     )
+    .command(
+      'supported-tokens <source> <router> <dest>',
```

Comment:
Can we use kamelCase for commands, for consistency?
I think yargs may allow both calls anyway